x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/job/retention/AbstractExpiredJobDataRemover.java

```diff
@@ -70,7 +69,7 @@ private void removeData(Iterator<Job> jobIterator, ActionListener<Boolean> liste
     }
 
     private long calcCutoffEpochMs(long retentionDays) {
-        long nowEpochMs = DateTime.now(ISOChronology.getInstance()).getMillis();
+        long nowEpochMs = Instant.now().toEpochMilli();
```

Comment:
This place is worthy to having a look too in regards to our comments about defaulted timezone